correlation_test.go

```diff
@@ -8,12 +8,13 @@ import (
 	"github.com/montanaflynn/stats"
 )
 
-func ExampleCorrelation() {
+func ExampleCorrelation(l *testing.T) {
```

Comment:
This fails for older Go versions which won't accept an argument in an en Example test:

https://travis-ci.org/github/montanaflynn/stats/builds/749863684